### ISSUE TITLE
[ACCESSIBILITY] Allowing customer to create inaccessible Color combinations

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#2043]
**Describe the accessibility issue**
Custom color editor allows customer to create inaccessible colors. 
This is low priority as this is only being used on a place (TO DO list) where user is making categories for themselves. 

**To Reproduce
Steps to reproduce the behavior:**

Go to '.Wellness.'
Click on '..To Do List.'
Scroll down to '..Manage Categories..'
Choose Custom color dropper icon and pick custom color (lighter the worse for accessibility)

**Expected behavior**
A Color contrast ratio of 4:5:1 should be met 

**Actual behavior**
Custom color editor allows customer to create inaccessible colors. 

**Screenshots**
https://photos.google.com/photo/AF1QipOMu7CoKOdlhSXHza4MpPP_rgNa4VNi4y52PWWp
https://photos.google.com/photo/AF1QipNJ2iXa5UKdKPgHqVVbIcHOX7LJfPGSI5w5yOTj

**Smartphone (please complete the following information):**
iPhone XR- iOS 14.6, Pixel 4a- Android 12

**Additional context**
This is low priority as this is only being used on a place (TO DO list) where user is making categories for themselves. HOWEVER, if this custom color choice is ever applicable to wider audience than self, then it should not allow inaccessible combinations.

**Comment by acovher2** [comment]
Found in 4.0.53

**Comment by acovher2** [comment]
Verified as fixed in 4.1.19
